pysat/instruments/pysat_testing2d.py

```diff
@@ -86,27 +87,25 @@ def load(fnames, tag=None, sat_id=None):
     # use that info to create a signal that is continuous from that start
     # going to presume there are 5820 seconds per orbit (97 minute period)
     time_delta = date - pysat.datetime(2009, 1, 1)
-    # root start
-    uts_root = np.mod(time_delta.total_seconds(), 5820)
     # mlt runs 0-24 each orbit.
-    mlt = np.mod(uts_root+np.arange(num)*scalar, 5820) * (24./5820.)
-    data['mlt'] = mlt
+    data['mlt'] = test.generate_fake_data(time_delta.total_seconds(),
+                                          np.arange(num)*scalar,
+                                          period=5820, data_range=24.0)
     # do slt, 20 second offset from mlt
-    uts_root = np.mod(time_delta.total_seconds() + 20, 5820)
-    data['slt'] = np.mod(uts_root + np.arange(num) * scalar,
-                         5820) * (24./5820.)
-
+    data['slt'] = test.generate_fake_data(time_delta.total_seconds()+20,
+                                          np.arange(num)*scalar,
+                                          period=5820, data_range=24.0)
     # create a fake longitude, resets every 6240 seconds
     # sat moves at 360/5820 deg/s, Earth rotates at 360/86400, takes extra time
     # to go around full longitude
-    long_uts_root = np.mod(time_delta.total_seconds(), 6240)
-    longitude = np.mod(long_uts_root + num_array, 6240) * (360. / 6240.)
-    data['longitude'] = longitude
-
+    data['longitude'] = test.generate_fake_data(time_delta.total_seconds(),
+                                                num_array, period=6240,
+                                                data_range=360.0)
```

Comment:
having data range be a single number seems like a bad idea.  Shouldn't it have a max and min?  E.g., data_range=[0.0, 360.0]

Reply:
Adding